test/config/MockAppConfig.scala

```diff
@@ -55,6 +55,8 @@ class MockAppConfig extends MockFactory {
     override lazy val timeoutDialogTimeout: Int = 900
 
     override lazy val taxYearErrorFeature: Boolean = true
+
+    override lazy val welshToggleEnabled: Boolean = true
```

Comment:
Is it worth having tests for when the toggle is disabled?

Reply:
As discussed, not needed for now